packages/trader/src/App/Components/Elements/ContractAudit/contract-details.tsx

```diff
@@ -172,6 +187,32 @@ const ContractDetails = ({
                                 ))}
                             </React.Fragment>
                         )}
+                        {isResetContract(contract_type) && (
+                            <React.Fragment>
+                                <ContractAuditItem
+                                    id='dt_bt_label'
+                                    icon={<Icon icon='IcContractStrike' size={24} />}
+                                    label={getBarrierLabel(contract_info)}
+                                    value={addCommaToNumber(Number(entry_spot_display_value)) || ' - '}
+                                />
+                                {reset_time && (
```

Comment:
Reset time and Reset barrier should be shown in contract audit only when reset_time start to come from BE (+-half of the contract)